SDPi_Supplement/asciidoc/volume2/dev-25/tf2-ch-a-mdpws-dev-25.adoc

```diff
@@ -0,0 +1,46 @@
+:var_uri_ws_discovery_transfer: https://www.w3.org/Submission/2006/SUBM-WS-Transfer-20060927/#Get
+:var_uri_dpws: http://docs.oasis-open.org/ws-dd/dpws/wsdd-dpws-1.1-spec.html
+[#vol2-clause-appendix-mdpws-dev-25]
+==== MDPWS: Discover BICEPS Services [DEV-25]
+
+===== GetMetadata Message
+
+The GetMetadata message is encoded by using {var_uri_ws_discovery_transfer}[WS-Get].
+
+====== Referenced Standards
+
+* <<ws-transfer-2006>> {var_uri_ws_discovery_transfer}[Section 3.1 Get]

```

Comment:
```suggestion
* <<ws_transfer_2006>> {var_uri_ws_discovery_transfer}[Section 3.1 Get]
```

Please also double check if the bib refs exist (see `SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc`)